.changes/v2.15.0/433-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* Fixes Issue #431 "Wrong order in Task structure" [GH-433]
+* Fixes Issue where VDC creation with storage profile `enabled=false` wasn't working
```

Comment:
Which VCD versions did you test this PR on? Also, which tests (all or just some tags) were ran?

Reply:
tag VDC, versions 10.2.2 and 10.3.1 - bug's and fixes doesn't depend on VCD